fe/projects/Seller.Portal/src/areas/Products/components/CategoryForm/CategoryForm.js

```diff
@@ -16,9 +16,10 @@ function CategoryForm(props) {
         name: { value: props.name ? props.name : "", error: "" },
         parentCategoryId: { value: props.parentCategoryId ? props.parentCategoryId : "" },
         files: { value: props.files ? props.files : [] },
-        schema: { value: props.schema ? JSON.parse(props.schema) : null },
+        schema: { value: props.schemas ? props.schemas.find(schema => schema.language === props.language) 
```

Comment:
**suggestion:** If schema is not used. Please remove and only send schemas in requestPayload